File: src/main/java/org/poglebiarka/app/PublicEndpoint.java
package org.poglebiarka.app;

import java.lang.annotation.Retention;
import java.lang.annotation.Target;

import static java.lang.annotation.ElementType.METHOD;
import static java.lang.annotation.RetentionPolicy.SOURCE;

/**
 * Marker only for showing which metod will be used for generate client call from frontend
 */
@Retention(SOURCE)
@Target(METHOD)
public @interface PublicEndpoint {
}

File: src/main/kotlin/org/poglebiarka/admin/PublicEndpoint.kt
package org.poglebiarka.admin;

import kotlin.annotation.AnnotationRetention.SOURCE
import kotlin.annotation.AnnotationTarget.FUNCTION


/**
 * Marker only for showing which metod will be used for generate client call from frontend
 */
@Retention(SOURCE)
@Target(FUNCTION)
annotation class PublicEndpoint
